server/opts.go

```diff
@@ -225,10 +230,10 @@ type Options struct {
 	MaxTracedMsgLen int `json:"-"`
 
 	// Operating a trusted NATS server
-	TrustedKeys              []string              `json:"-"`
-	TrustedOperators         []*jwt.OperatorClaims `json:"-"`
-	AccountResolver          AccountResolver       `json:"-"`
-	AccountResolverTLSConfig *tls.Config           `json:"-"`
+	TrustedKeys              []string             `json:"-"`
+	TrustedOperators         []*MergedClaimAndJWT `json:"-"`
```

Comment:
See previous comment. We can't do that. You would need to leave as-is and add a new option.
So looking at the code it seems that during the parsing of `operator` option, we get a list of files, then we call ReadOperatorJWT() on those to create the TrustedOperators slice. After option parsing, we would not have the list of files so we would not be able to get the jwt.
What we could do is either add a new option, deprecate this one (but keep it for ever, just at one point it may not have any content since it would be superseded by the new one), or we could have a private option (not exported) so we can build both in the configuration parsing.
But it seems that what you "expose" in the monitoring is just the JWT and so I am not sure that we need to have the option containing the merge of the two.
We can discuss all this since I may not make too much sense here :-)

Reply:
fixed